docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
+---
+
+The `fx` extension executes an Edge Worker.
+
+## Syntax
+
+Executes parameterless function:
+
+    <OBJECT> fx(<STRING> name)
+
+Executes function with parameters set via JSON or object:
+
+    <OBJECT> fx(<STRING> name, <STRING|OBJECT> json)
+
+Executes function with parameters set as pairs of arguments:
+
+    <OBJECT> fx(<STRING> name, <STRING> param1Name, <String|Bool|Int|Long|Double|Float> param1Value, ...)
+
+Examples of function with different syntax:
```

Comment:
This example is great, but I have no idea what it is supposed to do. I see a name and an age and money, but what is going on with this information?

Reply:
I removed it as I moved the examples to another place.